contrib/zsh-completion/_packer

```diff
@@ -1,70 +1,66 @@
-#compdef packer
```

Comment:
hmm, I wasn't able to get it to work with

```
rm ~/.zcompdump*
fpath=(./contrib/zsh-completion/ $fpath)
autoload -U compinit
compinit
zstyle ':completion:*' menu select=2
```

but the previous version did work